x-pack/plugins/ml/server/models/results_service/anomaly_charts.ts

```diff
@@ -766,12 +766,12 @@ export function anomalyChartsDataProvider(mlClient: MlClient, client: IScopedClu
     }
 
     // Build the tooltip data for the chart info icon, showing further details on what is being plotted.
-    let functionLabel = `${config.metricFunction}`;
+    let functionLabel = `${fullSeriesConfig.metricFunction ?? config.metricFunction}`;
```

Comment:
Can you find where the label for the SMV chart is being built? It is always showing `avg`:

<img width="536" alt="image" src="https://github.com/elastic/kibana/assets/7405507/38fa92ab-37e2-47da-8242-590e658a199d">